releases.yml

```diff
@@ -1,4 +1,27 @@
 releases:
+  art10187:
+    assembly:
+      basis:
+        assembly: 4.14.13
+        "reference_releases!": {}
+      type: custom
+      group:
+        advisories:
+          extras: 127885
+          image: -1
+          metadata: 127887
+          microshift: 127888
+          rpm: 127889
+        release_jira: ART-8940
+      members:
+        images:
+        - distgit_key: ose-cluster-ingress-operator
+          why: address OCPBUGS-36555
+          metadata:
+            labels:
+              "com.redhat.hotfix": OCPBUGS-36555
```

Comment:
This extra label may not take effect unless we rebuild the bundle or the operator image.

Reply:
updated new build with label